static/_headers

```diff
@@ -1,5 +1,5 @@
 /*
-  Content-Security-Policy: connect-src 'self' *.googleapis.com *.doubleclick.net https://status.maxmind.com https://www.maxmind.com https://api.hubspot.com https://forms.hscollectedforms.net https://forms.hsforms.com https://*.google-analytics.com https://*.analytics.google.com https://*.googletagmanager.com https://*.g.doubleclick.net https://*.google.com; default-src 'self'; font-src 'self' fonts.gstatic.com; form-action 'self' *.paypal.com; frame-ancestors 'self'; frame-src 'self' *.doubleclick.net *.paypal.com https://app.hubspot.com www.youtube.com; img-src 'self' data: https:; object-src 'none'; script-src 'self' 'report-sample' 'unsafe-inline'  https://js.hs-scripts.com https://js.hs-analytics.net https://js.hscollectedforms.net https://js.hs-banner.com https://js.usemessages.com https://www.maxmind.com *.googleapis.com www.youtube.com https://www.googleadservices.com https://www.google.com https://*.googletagmanager.com; style-src 'self' 'unsafe-inline' *.googleapis.com
+  Content-Security-Policy: connect-src 'self' *.googleapis.com *.doubleclick.net https://status.maxmind.com https://www.maxmind.com https://api.hubspot.com https://forms.hscollectedforms.net https://forms.hsforms.com https://*.google-analytics.com https://*.analytics.google.com https://*.googletagmanager.com https://*.g.doubleclick.net https://*.google.com; default-src 'self'; font-src 'self' fonts.gstatic.com; form-action 'self' *.paypal.com; frame-ancestors 'self'; frame-src 'self' *.doubleclick.net *.paypal.com https://app.hubspot.com https://www.google.com www.youtube.com; img-src 'self' data: https:; object-src 'none'; script-src 'self' 'report-sample' 'unsafe-inline'  https://js.hs-scripts.com https://js.hs-analytics.net https://js.hscollectedforms.net https://js.hs-banner.com https://js.usemessages.com https://www.maxmind.com *.googleapis.com https://cloud.google.com https://www.gstatic.com www.youtube.com https://www.googleadservices.com https://www.google.com https://*.googletagmanager.com; style-src 'self' 'unsafe-inline' *.googleapis.com www.gstatic.com
```

Comment:
Any reason the `www.gstatic.com` element is without https:// as it is on the dev site? Should we also add that to the *.googleapis.com option?